src/mlpack/tests/feedforward_network_test.cpp

```diff
@@ -685,10 +685,10 @@ BOOST_AUTO_TEST_CASE(RBFNetworkTest)
 
   FFN<NegativeLogLikelihood<> > model1;
   model1.Add<RBF<> >(dataset.n_rows, 10, centroids1);
-  model.Add<Linear<> >(10, 2);
+  model1.Add<Linear<> >(10, 2);
```

Comment:
Nice catch!

Reply:
I think 20 iteration was also helpful let's see now. What will happen now thanks for helping @zoq .